docs/source/sci_api_tutorial/armer.rst

```diff
@@ -2,21 +2,297 @@
+
+.. dropdown:: :fa:`eye,mr-1` User Interaction via Configured API`
+
+    For users, the interaction with ARMer is streamlined through configurations exposed by API developers:
+
+    Cluster Job Configuration Dictionary: This dictionary (cluster_job_config) shown above is provided by the API developers and exposes various configurable options that users can set according to their specific requirements. It includes parameters such as cluster type, environmental settings, and resource keywords.
+    
+    Simplified Job Submission: Users don’t need to manage complex cluster configurations directly. Instead, they provide necessary parameters through a high-level interface, simplifying the computational aspects of enzyme modeling.
+
+
+Input
+------------------------------------------------
+    In EnzyHTP, the cluster is an object that represents a specific HPC cluster configuration. This object is usually an instance of a class that implements the ClusterInterface or a similar interface that ARMer can interact with.
+    The input for ARMer should be all kinds of "clusters", for example, clusters for MD sampling, clusters for QM caluclations, etc.
+
+.. dropdown:: :fa:`eye,mr-1` Click here to see example use
+
+    .. admonition:: Example: how cluster_job_config is used in MD simulation sampling
+    (See Example Code for more examples)
+
+    .. code:: python
+
+        #Building MD Parameterization:
+        param_method = interface.amber.build_md_parameterizer(
+            ncaa_param_lib_path=f"{DATA_DIR}ncaa_lib",
+            force_fields=[
+                "leaprc.protein.ff14SB",
+                "leaprc.gaff",
+                "leaprc.water.tip3p",
+            ],
+        )
+
+         #Preparing Constraints:
+        mut_constraints = []
+        for cons in md_constraints:
+            mut_constraints.append(cons(mutant_stru))
+
+        #Configuring Cluster Job Settings: Sets up the configuration for the job submission to the cluster, specifically tailoring it for GPU-intensive tasks required in MD simulations.
+        md_cluster_job_config = {
+            "cluster" : Accre(),
+            "res_keywords" : {
+                "account" : "csb_gpu_acc",
+                "partition" : "pascal"
+            }
+        }
+        
+        # Executes the equilibrium molecular dynamics sampling using the specified structure and parameters.
+        md_result = equi_md_sampling(
+            stru = mutant_stru,
+            param_method = param_method,
+            cluster_job_config = md_cluster_job_config, #The job configuration for submitting this task to the cluster.
+            job_check_period=10,
+            prod_constrain=mut_constraints,
+            prod_time=md_length,
+            record_period=md_length*0.1,
+            work_dir=f"{mutant_dir}/MD/"
+        )        
+
+Example Code
+==============================================
+
+Calculate single point energy for a QM cluster
```

Comment:
Please make this example more focused on teaching user how to prepare/use the cluster_job_config.

We already have a QM cluster tutorial so no need to explain anything related to that (you can just add a link to the page)